www/common/translations/messages.js

```diff
@@ -1133,7 +1133,7 @@ define(function () {
     out.help.title = "Getting started";
     out.help.generic = {
         more: 'Learn more about how CryptPad can work for you by reading our <a href="/faq.html" target="_blank">FAQ</a>',
-        share: 'Use the share menu (<span class="fa fa-shhare-alt"></span>) to generate a link so collaborators can view or edit this pad',
+        share: 'Use the share menu (<span class="fa fa-share-alt"></span>) to generate a link so collaborators can view or edit this pad',
```

Comment:
This looks like a typo, but the double-h is deliberate. Various adblockers remove any element with classes including `fa-share`.